apple-pie/styles/style.css

```
@@ -0,0 +1,95 @@
+.container{
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    font-family: 'Roboto', sans-serif;
+    height: 1000px;
+    width: 1520px;
+
+}
+
+.applepic{
+    height: 1000px;
+    width: 1450px;
+   
+}
+
+.applepie{
```

solo sirve para contenderores
```suggestion
.applepie{
color: white;
font-size: 100px;
}
```